server/publication.js

```diff
@@ -1,5 +1,5 @@
 Meteor.publish('projects', function() {
-    return Projects.find({creatorId: this.userId}) ;
+    return Projects.find( { $or: [ { creatorId: this.userId } , { userId: null } ] } );
```

Comment:
I didn't have time to check this yet, but shouldn't the second or part be `{ creatorId: null }` ? Because what is the userId in this context?